opal/mca/accelerator/cuda/accelerator_cuda.c

```diff
@@ -498,13 +498,14 @@ static int accelerator_cuda_memcpy(int dest_dev_id, int src_dev_id, void *dest,
      * Additionally, cuMemcpy is not necessarily always synchronous. See:
      * https://docs.nvidia.com/cuda/cuda-driver-api/api-sync-behavior.html
      * TODO: Add optimizations for type field */
-    result = cuMemcpyAsync((CUdeviceptr) dest, (CUdeviceptr) src, size, GET_STREAM(opal_accelerator_cuda_memcpy_stream.super));
+    result = cuMemcpyAsync((CUdeviceptr) dest, (CUdeviceptr) src, size,
+                           GET_STREAM(&opal_accelerator_cuda_memcpy_stream.base));
```

Comment:
@devreal Does this change do what you intended it for?

Reply:
why do you need the GET_STREAM macro in this particular scenario?  I think you can just use the opal_accelerator_cuda_memcpy_stream.stream here, there is no ambiguity with the default stream (which is what the GET_STREAM macro was designed to do), isn't it?

Reply:
@edgargabriel I think you are right. Updated the PR.